src/main/java/frc/robot/Robot.java

```diff
@@ -171,14 +181,14 @@ public void testInit() {
         CommandScheduler.getInstance().cancelAll();
     }
 
-    public CommandBase regurgitateFC() {
-        CommandBase cmd = new ParallelCommandGroup(intake.discharge(), indexer.reversePush());
-        cmd.setName("Regurgitate FC");
+    public CommandBase regurgitate() {
+        CommandBase cmd = new ParallelCommandGroup(intake.discharge(), indexer.discharge());
+        cmd.setName("Regurgitate");
         return cmd;
     }
 
     public CommandBase shootAuto() {
-        CommandBase cmd = new SequentialCommandGroup(releaseBalls(3), shooter.spinDown(), drive.driveDistance(20, .5));
+        CommandBase cmd = new SequentialCommandGroup(releaseBalls(3), shooter.spinDown(), drive.driveDistance(40, .5));
```

Comment:
This is the second time I've seen this exact drive command. Can we make a factory function for it?